packages/frontend/src/hooks/useOrganizationUsers.ts

```diff
@@ -86,20 +96,64 @@ export const usePaginatedOrganizationUsers = (params: {
     return useQuery<ApiOrganizationMemberProfiles['results'], ApiError>({
         queryKey: [
             'organization_users',
-            params.includeGroups,
-            params.paginateArgs,
-            params.searchInput,
+            includeGroups,
+            paginateArgs,
+            searchInput,
         ],
         queryFn: () =>
-            getOrganizationUsersQuery(
-                params.includeGroups,
-                params.paginateArgs,
-                params.searchInput,
-            ),
+            getOrganizationUsersQuery(includeGroups, paginateArgs, searchInput),
         onError: (result) => setErrorResponse(result),
     });
 };
 
+export const useInfiniteOrganizationUsers = (
+    {
+        searchInput,
+        includeGroups,
+        pageSize,
+    }: {
+        searchInput?: string;
+        includeGroups?: number;
+        pageSize: number;
+    },
+    infinityQueryOpts: UseInfiniteQueryOptions<
+        ApiOrganizationMemberProfiles['results'],
+        ApiError
+    > = {},
+) => {
+    const setErrorResponse = useQueryError();
```

Comment:
What does this do?

Reply:
This is a hook that processes error responses and shows the toast depending on the error 👌